client/version.go

```diff
@@ -0,0 +1,18 @@
+package client
+
+import "fmt"
+
+const versionString = "1.2.3"
```

Comment:
Let's make this a string that defaults to devel or something similar and add some documentation on how to override it at build time using ldflags.